src/autograd.cpp

```diff
@@ -26,15 +44,15 @@ void cpp_autograd_set_grad_mode (bool enabled) {
 // [[Rcpp::export]]
 bool cpp_autograd_is_enabled()
 {
-  return lantern_autograd_is_enabled();  
```

Comment:
Please ignore whitespace changes for now. Sorry.

You can click the gear icon on top, and then tick the "Hide whitespace" check box, and then "apply and reload".